code/modules/projectiles/updated_projectiles/ammunition.dm

```diff
@@ -105,7 +105,7 @@ They're all essentially identical when it comes to getting the job done.
 		return
 
 	var/obj/item/ammo_magazine/handful/new_handful = new /obj/item/ammo_magazine/handful()
-	var/MR = caliber == "12g" ? 5 : 8
+	var/MR = (caliber in list("12g", "7.62x54mmR")) ? 5 : 8
```

Comment:
this is fucking awful

Reply:
@Ghommie why is this better than what I had before.

Reply:
It's not so much better than before, either ways look awful. The ammo calibers should be defines, and that list should be a define too, but I didn't pretend from you to do that.